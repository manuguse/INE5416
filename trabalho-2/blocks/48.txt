1 1 8 12 16 16 16 23 27 27 27 27
1 8 8 8 16 16 18 24 24 30 27 27
2 2 2 2 13 18 18 18 24 30 32 32
3 3 2 13 13 19 19 18 24 24 33 32
4 4 2 14 14 19 19 25 25 25 33 33
5 6 6 14 14 19 19 25 25 25 34 33
5 6 9 14 14 19 21 21 25 21 34 34
6 6 9 14 17 17 22 21 21 21 34 34
7 6 9 15 15 17 17 17 28 28 35 35
7 9 9 15 15 17 20 20 26 31 35 35
7 7 7 7 15 20 20 26 26 31 31 31
7 10 11 11 11 20 20 20 29 31 31 31