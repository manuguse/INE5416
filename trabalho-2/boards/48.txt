3 - - - 4 - 5 - - 3 - 6
- 4 - - 2 - 4 - - - - 2
3 - 6 - - - - - 2 - - -
- - - 2 1 - - 2 - - - 1
- - - - - - 2 5 3 7 - 2
- - - 6 - 4 - - 2 - - -
- 4 - - 1 - - - 1 - - 3
6 - - 2 - 5 - 5 3 - - -
- - - - - - 3 6 1 - - 4
- - - - 3 - - 7 - 6 - 1
- 7 - - - - 5 - - 5 7 -
- - - - - - 3 - - 3 - 2