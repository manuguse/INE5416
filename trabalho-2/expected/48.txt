3 1 2 1 4 3 5 1 5 3 4 6
2 4 1 3 2 1 4 5 4 2 1 2
3 2 6 1 3 5 1 3 2 1 2 3
2 1 5 2 1 7 5 2 1 3 4 1
1 2 4 7 5 6 2 5 3 7 3 2
2 5 1 6 3 4 1 4 2 6 5 1
1 4 5 4 1 3 4 6 1 2 4 3
6 3 4 2 4 5 1 5 3 1 2 1
5 2 3 4 5 2 3 6 1 2 3 4
4 1 2 1 3 1 6 7 3 6 2 1
3 7 1 6 2 4 5 1 2 5 7 4
2 1 2 1 3 1 3 2 1 3 1 2